components/consent/consent.css

```diff
@@ -34,30 +34,30 @@ amp-consent .ampstart-btn {
   .ampstart-consent-content {
     flex-flow: column wrap;
     align-items: center;
-    margin: 12px 25px 15px 12px;
+    margin: .75rem var(--space-2) var(--space-1) .75rem;
   }
 
   .ampstart-consent-popup p {
-    padding: 5px 20px 5px 10px;
-    margin: 0 0 10px 0;
+    padding: .25rem 1.25rem .25rem .625rem;
```

Comment:
https://github.com/ampproject/ampstart/blob/master/css/ampstart-base/bass-override-vars.css#L25

You should use one of these especially for margins/padding/outlines 

since the component could get used in different templates and each template will define what their spacing theme is, you should use the spacing vars.